src/mixins/filing-template-mixin.ts

```diff
@@ -829,6 +831,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     this.setEntityType(draftFiling.business.legalType || this.getBusinessLegalType)
     this.setLegalName(draftFiling.business.legalName || this.getBusinessLegalName)
     this.setFoundingDate(draftFiling.business.foundingDate || this.getBusinessFoundingDate)
+    this.setBusinessStartDate(this.getBusinessStartDate)
```

Comment:
Why not save it to, and restore it from, `draftFiling.business.startDate` (same as other business properties)?

Reply:
`startDate` is not returned in the response of `fetchFirstTask` legal api call. Hence, setting it in business Info

![image](https://github.com/bcgov/business-create-ui/assets/116035339/ad9846d7-52a3-4e8d-ad43-314e9236d01f)

![image](https://github.com/bcgov/business-create-ui/assets/116035339/7335ccd7-36c7-47ec-aacf-43d05cb2fb4e)

Reply:
Ah, I see.

This "fetch first task" thing is kludgey anyway. I would be better to pass in the id of the filing, but that's a future problem.

Wouldn't it be much cleaner to update Filings UI so it populates Start Date in the draft?

Reply:
Yes. I will change Edit UI and create a PR soon

Reply:
This is resolved. I assume you'll test this and it will be fine :)